paseos/attitude/geometric_model.py

```diff
@@ -0,0 +1,123 @@
+from loguru import logger
+import numpy as np
+import trimesh
+
+
+class CuboidGeometricModel:
+    """This model describes the geometry of the spacecraft
+    Currently it assumes the spacecraft to be a cuboid shape, with width, length and height
+    """
+
+    _actor = None
+    _actor_mass = None
```

Comment:
already exists https://github.com/aidotse/PASEOS/blob/e5bb071b4ee2988c2765497aa8fe3d0f6eacb655/paseos/actors/spacecraft_actor.py#L20

Reply:
I've deleted line 12 but I'm not entirely sure whether I should change anything else